### PR TITLE
Update enhanced RDS templates and README

### PR DESCRIPTION
**Pull request by trevoroc**
Breaks the template into two to offer both
encrypted and non-encrypted options for access
tokens.

### CODE REVIEW

File: enhancedRDS/README.md [hunk 1]
```diff
@@ -1,56 +1,147 @@
...
-Decrypt permissions in addition to the Lambda execution permissions. 
-
-Once you have identified or created the execution role, save the ARN of that
-role (this can be found on the IAM Role page for that role). This will be a
-parameter for the template.
-
-### Encrypt your access token
+
+### Either deployment method
+Choose security preferences regarding your SignalFx access token.
```
Comment (beccatortell) [inline enhancedRDS/README.md]: Seems like the "choose security preferences" para is meant to go here as a first step.

File: enhancedRDS/README.md [hunk 2]
```diff
@@ -1,56 +1,147 @@
...
+[here](http://docs.aws.amazon.com/cli/latest/reference/kms/encrypt.html). Make
+sure you have access to the cipher text output by the encryption as well as the
+key id of the encryption key you used.
+
+### Determine the desired set of metric groups
+This step is necessary if you only want a subset of the metric groups to be
+reported. You can find documentation on the available metrics
+[here](http://docs.aws.amazon.com/AmazonRDS/latest/UserGuid/USER_Monitoring.OS.html).
+
+The available groups are as follows:
```
Comment (beccatortell) [inline enhancedRDS/README.md]: Once I've decided on a subset how do I ingest just that subset?

File: enhancedRDS/README.md (hunk 1 above)
Comment (trevoroc) [reply enhancedRDS/README.md]: That was the intention, should we just move that security preferences paragraph up to replace this?

File: enhancedRDS/README.md (hunk 2 above)
Comment (trevoroc) [reply enhancedRDS/README.md]: I thought it made sense to leave that for the stage when it actually gets input, but that part probably wasn't clear enough. Should I move that up here or is it fine under "Fill out application parameters"?

**Pull request merged**